### PR TITLE
docs: fix router base

### LINKED ISSUE #165: Documentation navigation: URLs for "On this page" go to the starting page instead of the respective section
*To reproduce:* Visit for example [Relationship Operations](https://robsontenorio.github.io/vue-api-query/api/relationship-operations/).

*Expected behaviour:* Opening links for "On this page" in a new window (on the right in widescreen layout) should link to their respective sections in the documentation.  
Linking to specific headlines in the documentation is especially useful so we can send people to a specific section of a documentation page.

*What actually happens:* The navigation only works when left clicking. Opening the resulting URL in a new browser window leads back to the documentation's starting page. The underlying links are not working correctly.  
Additionally, hovering over a headline and clicking the `#` also links back to the documentation's starting page.

### PR DESCRIPTION
**Pull request by JoaoPedroAS51**
Temporary workaround to solve router base issue. See #165 and https://github.com/nuxt/content/issues/376#issuecomment-702193217 for more information.

I tested and couldn't find any side effects. As the docs is a basic website, I believe we will not have any side effects.

Fixes #165

### CODE REVIEW

File: docs/nuxt.config.js [hunk 1]
```diff
@@ -3,5 +3,10 @@ import theme from "@nuxt/content-theme-docs";
 export default theme({
   router: {
     base: '/vue-api-query/'
+  },
+  hooks: {
+    'vue-renderer:ssr:templateParams': function(params) {
+      params.HEAD = params.HEAD.replace('<base href="/vue-api-query/">', '')
+    }
```
Comment (robsontenorio) [inline docs/nuxt.config.js:10]: @JoaoPedroAS51 Mabye some comment here pointing into nuxt issue, for future final fixing.

**Review by robsontenorio — CHANGES_REQUESTED** [review]

File: docs/nuxt.config.js (hunk 1 above)
Comment (JoaoPedroAS51) [reply docs/nuxt.config.js:10]: Good idea!

Comment (JoaoPedroAS51) [reply docs/nuxt.config.js:10]: Just note that the nuxt issue I mentioned is currently closed.

**Review by robsontenorio — APPROVED** [review]

**Comment by robsontenorio** [comment]
@JoaoPedroAS51 i think it is good to merge immediately.

**Pull request merged**